Makefile

```diff
@@ -9,7 +9,10 @@ precommit: test.dev
 install:
 	go mod tidy
 	npm install
-	(cd livebud && npm install)
+# Note: We don't use a subshell here to support Windows. See #79.
```

Comment:
This should be tabbed and be prefixed with `@ ` so it doesn't show up when you run `make install`